intelmq/tests/test_conf.py

```diff
@@ -27,6 +27,7 @@
 from intelmq.lib.utils import lazy_int
 
 yaml = YAML(typ="safe", pure=True)
+yaml.default_flow_style = False
```

Comment:
Can you please add an inline comment why this is necessary/useful?

Reply:
Fixed.